Gemfile

```diff
@@ -19,15 +19,14 @@ gem "globalize", "~> 5"
 gem "govuk_ab_testing", "~> 2.4x"
 gem "govuk_admin_template", "~> 6.7"
 gem "govuk_app_config", "~> 2.0"
-gem "govuk_publishing_components", "~> 21.11.0"
+gem "govuk_publishing_components", "~> 21.3.0"
```

Comment:
It needs at least [`21.11.0`](https://github.com/alphagov/govuk_publishing_components/blob/master/CHANGELOG.md#21130), as the organisation logos need to be wrapped in a heading rather than a `div`